cardano-cli/src/Cardano/CLI/Shelley/Parsers.hs

```diff
@@ -412,6 +417,32 @@ pKeyCmd =
           <> Opt.completer (Opt.bashCompleter "file")
           )
 
+    pKeyConvertCardanoAddressSigningKey :: Parser KeyCmd
+    pKeyConvertCardanoAddressSigningKey =
+      KeyConvertCardanoAddressSigningKey
+        <$> pCardanoAddressKeyType
+        <*> pSigningKeyFile Input
```

Comment:
Perhaps I should use an option parser whose help text specifically denotes that we expect a file containing a Bech32-encoded extended signing key.

Reply:
Seems a good idea.